<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>651</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>801</width>
      <height>559</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="fileInputButton">
         <property name="text">
          <string>Загрузить файл</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QPushButton" name="scaleToLessButton">
           <property name="text">
            <string>меньше</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Изменить масштаб</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="scaleToMoreButton">
           <property name="text">
            <string>больше</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QSlider" name="scaleSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="moveXDoubleSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="moveYDoubleSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="moveZDoubleSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Перемещение</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>y</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_12">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>z</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spinYdoubleSpinBox"/>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="spinZdoubleSpinBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinXdoubleSpinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Поворот</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_15">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Толщина ребер</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="verticesThiknessLessButton">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="linesThiknessLessButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>меньше</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="linesThiknessMoreButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>больше</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QSlider" name="linesThiknessSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Толщина вершин</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="verticesThiknessMoreButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QSlider" name="verticesThiknessSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="glview" name="myglwidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Сделать гифку</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="gifButton">
         <property name="text">
          <string>Бам</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="screenshotButton">
         <property name="text">
          <string>Бум</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Сделать скрин</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="objnamelabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Количество вершин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="verticesLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Количество ребер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="indicesLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="rotateXcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate X 90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>580</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rotateYcheckBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>580</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Z 90</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rotateZcheckBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Y 90</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <widget class="QMenu" name="menuSet_projection_type">
     <property name="title">
      <string>Set projection type</string>
     </property>
     <addaction name="action1"/>
     <addaction name="action2"/>
    </widget>
    <widget class="QMenu" name="menuEdges">
     <property name="title">
      <string>Edges</string>
     </property>
     <addaction name="actionsolid"/>
     <addaction name="actiondashed"/>
     <addaction name="separator"/>
     <addaction name="actioncolor"/>
    </widget>
    <widget class="QMenu" name="menuVertices">
     <property name="title">
      <string>Vertices</string>
     </property>
     <addaction name="actionNone"/>
     <addaction name="actionCircle"/>
     <addaction name="actionSquare"/>
     <addaction name="separator"/>
     <addaction name="actionSet_color"/>
    </widget>
    <widget class="QMenu" name="menuScrinshot_type">
     <property name="title">
      <string>Scrinshot type</string>
     </property>
     <addaction name="action_jpg"/>
     <addaction name="action_bpm"/>
    </widget>
    <addaction name="actionSet_background_color"/>
    <addaction name="menuSet_projection_type"/>
    <addaction name="menuEdges"/>
    <addaction name="menuVertices"/>
    <addaction name="menuScrinshot_type"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_background_color">
   <property name="text">
    <string>Set background color</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>parallel</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>central</string>
   </property>
  </action>
  <action name="actionsolid">
   <property name="text">
    <string>solid</string>
   </property>
  </action>
  <action name="actiondashed">
   <property name="text">
    <string>dashed</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="text">
    <string>color</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionSet_color">
   <property name="text">
    <string>Set color</string>
   </property>
  </action>
  <action name="action_jpg">
   <property name="text">
    <string>.jpg</string>
   </property>
  </action>
  <action name="action_bpm">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>.bmp</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glview</class>
   <extends>QWidget</extends>
   <header location="global">view/glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
